packaging/ubuntu-14.04/rules

```diff
@@ -174,6 +172,7 @@ override_dh_install:
 	# it's a conf file so we're stuck with it
 	mv debian/snapd/etc/profile.d/snapd.sh debian/snapd/etc/profile.d/apps-bin-path.sh
 
+	# trusty needs this to make /snap rshared
 	install --mode=0644 debian/snap.mount.service debian/snapd/$(SYSTEMD_UNITS_DESTDIR)
```

Comment:
Hmmm, hold on, didn't we do that from snap-confine and de-comissioned this unit file?

Reply:
no idea. Did we?

Reply:
We do this from snap-confine now, but I'm not sure we can de-commission the unit file just yet. We still have this problem that snap-confine runs for the first time *after* all the snaps are mounted (we still have this problem on 16.04 and its causing trouble with lxd).